packages/react-scripts/template/README.md

```diff
@@ -493,6 +494,50 @@ In development, expressing dependencies this way allows your styles to be reload
 
 If you are concerned about using Webpack-specific semantics, you can put all your CSS right into `src/index.css`. It would still be imported from `src/index.js`, but you could always remove that import if you later migrate to a different build tool.
 
+## Adding a CSS Modules stylesheet
+
+This project supports [CSS Modules](https://github.com/css-modules/css-modules) alongside regular stylesheets using the **[name].module.css** file naming convention. CSS Modules allows the scoping of CSS by automatically prefixing class names with a unique name and hash.
```

Comment:
super nitpicky: 

Since the prefix is now done by directory and filename, do you want to update this line to say that and also mention the benefits (as noted in your comment https://github.com/facebookincubator/create-react-app/pull/2285#issuecomment-340401841 )

Or I could do that for you if you're too busy.

Reply:
Thanks for finding that. Updated :)